en/components/grid.md

```diff
@@ -639,3 +638,17 @@ Our community is active and always welcoming to new ideas.
 
 * [Ignite UI for Angular **Forums**](https://www.infragistics.com/community/forums/f/ignite-ui-for-angular)
 * [Ignite UI for Angular **GitHub**](https://github.com/IgniteUI/igniteui-angular)
+
+## Grid Limitations
+
+<div ckass="devider"></div>
+
+### General
+
+|Limitation|Description|
+|--- |--- |--- |
+|Column widths into `percentage`|Currently we do not support mixing column widths with `%` and `px`.
+|Skipped `height` on the parent of the grid or grid itself| The `grid` is not rendered correctly whenever the `height` on the parent or on the grid itself is omitted.|
+|`NaN` whenever filter number column|When filter number column and enter character different from digit, `NaN` value appers due incorrect cast and no filter is applied.|
+|`undefined` in edit mode|When `data` persists `undefined` values and enter into `edit mode` the string value `undefined` is displayed.|
+|Grid `width` |Currently the `width` of the grid is spanning the container(parent) it is placed in. Column `widths` does not reflect on the `view port` of the grid.|
```

Comment:
Grid `width` does not depend on the column widths | The width of all columns does not determine the spaning of the grid itself, it is determined by the parent container dimensions or the difined grid's width.

Grid nested in parent container | When grid's width is not set and it is placed in parent container with defined dimensions, the grid spans to this container.